### PR TITLE
Fixed the zombie_cluster global resources

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description
1. Skip the deletion of IAM User, as it is risky permission.
2. If the zombie_cluster has only IAM Role/ S3 bucket, upload to elastic search only if ran in region=us-east-1

### CODE REVIEW

**Review by ebattat — COMMENTED** [review]
@athiruma, did u test it locally ?

**Comment by athiruma** [comment]
> @athiruma, did u test it locally ?

yes, I did.

File: cloud_governance/policy/policy_operations/aws/zombie_cluster/run_zombie_cluster_resources.py [hunk 1]
```diff
@@ -54,8 +55,8 @@ def __get_resource_list(region, delete: bool = False, resource: str = '', cluste
                                     zombie_cluster_resources.zombie_cluster_subnet,
                                     zombie_cluster_resources.zombie_cluster_vpc
                                     ]
-    iam_zombie_resource_services = [zombie_cluster_resources.zombie_cluster_role,
-                                    zombie_cluster_resources.zombie_cluster_user]
+    iam_zombie_resource_services = [zombie_cluster_resources.zombie_cluster_role]
+    # [zombie_cluster_resources.zombie_cluster_user]
```
Comment (ebattat) [inline cloud_governance/policy/policy_operations/aws/zombie_cluster/run_zombie_cluster_resources.py:59]: Add comment

Comment (athiruma) [reply cloud_governance/policy/policy_operations/aws/zombie_cluster/run_zombie_cluster_resources.py:60]: done

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**